### DISCUSSION TITLE
[Craft 4] How to querying addresses in Graph QL?

### DISCUSSION THREAD

**Discussion by aodihis** [craftcms/cms#10882] [Q&A] [answered]
Hello, 
I have been playing around with Craft 4, and I found out there is a new elements called Address that attached to the User.
My question how to querying the address that the user saved, I am not yet found any way to get in through rest or the graphQL.
In graphQL I have been try this querying but also not work.

`{
	user(email:"admin@admin.com") {
		... on User {
			addresses
		}
	}
}`

it said `Cannot query field \"addresses\" on type \"Query\"."`, but I can querying another custom fields using this query.

**Comment by brandonkelly** [comment, marked as answer]
We haven’t added address support to GraphQL yet. Coming soon!

**Reply by brandonkelly** [reply]
Added now, for the next release (9f9bb38fdc89ca97eff78543d75602afa9f9e113) ✨

**Reply by aodihis** [reply]
Wow, Thanks @brandonkelly.
